docs/usage.md

```diff
@@ -140,6 +151,7 @@ Metadata table (TSV) is the "Sample_preparations_sheet.tsv" that can be directly
 - **QBiC Code**: is needed to match metadata with the raw counts.
 - **Secondary Name**, samples will be named with the pattern: QBiC code + Secondary name.
 - **Condition: tag**: a separated column for each of the conditions. The headers of this columns start with "Condition: ". The values of these columns should not contain spaces.
+- **Data ID**: is needed to match metadata with the rsem or salmon files (not necessary for featurecounts).
 
 ```tsv
 QBiC Code   Secondary Name  Condition: treatment
```

Comment:
Also give an example of a table with a Data ID if needed here

Reply:
Not necessary any more, deleted this entry